pkg/data/values_test.go

```diff
@@ -0,0 +1,108 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetValue(t *testing.T) {
+	tests := []struct {
```

Comment:
Nit: you may want to add a test case where the top-level `data` input is a slice, not a map.

Reply:
Passing in a slice as the top level is not supported, this would not compile unless we changed the function signature.

Reply:
You did already change the function signature, such that it accepts `interface{}`.

Reply:
Indeed I did and somehow forgot, adding the test